read-fonts/src/tables/gsub/closure.rs

```diff
@@ -0,0 +1,188 @@
+//! Computing the closure over a set of glyphs
+//!
+//! This means taking a set of glyphs and finding the set of glyphs that are
+//! reachable from those glyphs via substitution.
```

Comment:
It's arguably implicit implicit in the signature of closure_glyphs but I suggest you note that this means reachable by a given set of glyphs in any order, including many instances of any member of the set.